org.whatwg.awesome/style.css

```diff
@@ -90,7 +90,7 @@ h2 > a { display: block; border: 0; }
 header { margin:3em 0 3em 0; width: 100%; text-align: center; padding-bottom:0.25em; }
 header img { height:.8em }
 header a { border-bottom:none; color:#3c790a }
-.description:first-of-type { margin: 0; color: rgb(0,0,0); padding: 0; }
+.description:first-of-type { margin: 0; color: rgb(0,0,0); padding: 0; font-size: 1.333em; font-family: Rockwell, Georgia, serif; }

```

Comment:
Does `:first-of-type` mean there are multiple elements with `.description`? If so, perhaps we should target this `p` using `hgroup p`.

Reply:
There aren't, but I didn't want to mess with stuff too much in this PR. It's just a weird way of selecting the blog description.